Terms.md

```diff
@@ -36,19 +36,20 @@ Here is a summary of several key provisions of Sage Bionetworks’ Terms of Serv
 </ul>  
 
 <p><strong>When contributing User Content, or accessing advanced features you must: </strong></p>
-
-- Complete the Sage Bionetworks training to demonstrate understanding of rules for redistributing datasets, Software, or other Content, and interacting with cloud technologies, APIs, and de-identified information.
-- Remove direct HIPAA identifiers from Content you contribute, unless otherwise unambiguously authorized. See information about de-identification in accordance with the [HIPAA Privacy Rule](https://www.hhs.gov/hipaa/for-professionals/privacy/laws-regulations/index.html).
-- Ensure that data participants have consented to the sharing and use of their Data, where required by applicable law.
-- License Public Content under [Creative Commons BY 4.0](https://creativecommons.org/licenses/by/4.0/) and open-source licenses like [Apache 2.0.](https://www.apache.org/licenses/LICENSE-2.0)
-- Grant Sage broader licenses for Private Content to operate services, improve products, and train AI models.
+<ul>
+<li><p>Complete the Sage Bionetworks training to demonstrate understanding of rules for redistributing datasets, Software, or other Content, and interacting with cloud technologies, APIs, and de-identified information.</p></li>
+<li><p>Remove direct HIPAA identifiers from Content you contribute, unless otherwise unambiguously authorized. See information about de-identification in accordance with the [HIPAA Privacy Rule](https://www.hhs.gov/hipaa/for-professionals/privacy/laws-regulations/index.html).</p></li>
+<li><p>Ensure that data participants have consented to the sharing and use of their Data, where required by applicable law.</p></li>
+<li><p>License Public Content under [Creative Commons BY 4.0](https://creativecommons.org/licenses/by/4.0/) and open-source licenses like [Apache 2.0.](https://www.apache.org/licenses/LICENSE-2.0).</p></li>
+<li><p>Grant Sage broader licenses for Private Content to operate services, improve products, and train AI models.</p></li>
+</ul>
 
 <p><strong>We want you to be aware that:</strong></p>
 <ul>
 <li><p>We will remove Content if we deem it inappropriate and will cancel your account for violations, or prolonged inactivity.</p></li>
 <li><p>We respect the intellectual property rights, including copyrights, patents, trade secrets, and trademarks, of others, and ask you to do the same. You retain copyright in the content you post but grant licenses to Sage and other users.</p></li>
 <li><p>We have the right to use your Content to operate our services and develop new products.</p></li>
-<li><p>You must be 18 years or older to use our services.</p></li>
+<li><p>You must be at least 18 years old to use our services or you must obtain a parent/guardian authorization by completing a [Child Minor Addendum].</p></li>
```

Comment:
Need to link this [Child Minor Addendum]
Do you want to link to a page on www.synapse.org that will render this content?  If so, let's say it will be
https://www.synapse.org/TrustCenter:ChildMinorAddendum

Reply:
Perfect. I will also include the link to the ToS